portfolio/src/main/webapp/index.html

```diff
@@ -295,7 +280,7 @@ <h1 class="title">Contact me</h1>
                 <a href="https://github.com/erichli1" target="_blank" style="text-decoration: none;">
                     <i class="fab fa-github fa-2x"></i>
                 </a>
-            </div>
+            </div>        
```

Comment:
Extra whitespace at the end?

Reply:
Just committed a change to remove the whitespace!